packages/discord.js/src/structures/Entitlement.js

```diff
@@ -159,6 +169,16 @@ class Entitlement extends Base {
   fetchUser() {
     return this.client.users.fetch(this.userId);
   }
+
+  /**
+   * Marks this entitlement as consumed
+   * <info>Only available for One-Time Purchase consumable SKUs.</info>
+   * @returns {Promise<void>}
+   */
+  async consume() {
+    // Return this.client.rest.post(Routes.consumeEntitlement(this.applicationId, this.id));
```

Comment:
Is this still needed? Or was -types not yet updated with this 👀

Reply:
It is merged, but not yet released.

Reply:
Oh. Want me to release it now so we can merge it in?

Reply:
Sure, why not. Famous last words...

Reply:
Has been done